addons/godot-xr-tools/plugin.gd

```diff
@@ -84,6 +84,22 @@ func _enter_tree():
 			"0.2,0.8,0.05",
 			0.7)
 
+	# Add input y_axis_dead_zone to the project settings
+	_define_project_setting(
+			"godot_xr_tools/input/y_axis_dead_zone",
+			TYPE_FLOAT,
+			PROPERTY_HINT_RANGE,
+			"0.0,0.5,0.01",
+			0.1)
+
+	# Add input x_axis_dead_zone to the project settings
+	_define_project_setting(
+			"godot_xr_tools/input/x_axis_dead_zone",
+			TYPE_FLOAT,
+			PROPERTY_HINT_RANGE,
+			"0.0,0.5,0.01",
+			0.2)
```

Comment:
is it correct our default here is 0.2?

Reply:
when testing, I found useful adding a bit more deadzone in the x axis to avoid accidental rotation when moving forward... 

the original issue suggested a value of 0.2.  I'm not sure which value would be more convenient by default

Reply:
Owh, yeah that makes sense :)